ui/i18n/qml_en.ts

```diff
@@ -571,10 +571,10 @@
     </message>
 </context>
 <context>
-    <name>RenameAccontModal</name>
+    <name>RenameAccountModal</name>
     <message numerus="yes">
         <source>Account name must be at least %n character(s)</source>
-        <translation>
+        <translation type="unfinished">
```

Comment:
You can just remove the "unfinished" attribute :) Also there's a git conflict there